src/wovnio/html/HtmlConverter.php

```diff
@@ -0,0 +1,155 @@
+<?php
+namespace Wovnio\Html;
+
+/**
+ * Convert html via Simple HTML DOM Parser
+ *
+ * @see http://simplehtmldom.sourceforge.net/manual.htm
+ */
+class HtmlConverter {
+  private $html;
+  private $token;
+
+  /**
+   * HtmlConverter constructor.
+   * @param String $html
+   * @param String $token project_token
+   */
+  public function __construct($html, $token) {
+    $this->html = $html;
+    $this->token = $token;
+  }
+
+  /**
+   * Convert to appropriate HTML to send to Translation API
+   * e.g.) remove wovn-ignore content
+   *
+   * @return array converted html and HtmlReplaceMarker
+   */
+  public function convertToAppropriateForApiBody() {
+    // encoding detection uses 30% of execution time for this method, should we make encoding as setting?
+    $encoding = mb_detect_encoding($this->html, array('UTF-8', 'EUC-JP', 'SJIS', 'eucJP-win', 'SJIS-win', 'JIS', 'ISO-2022-JP', 'ASCII'));
+    $dom = str_get_html($this->html, false, false, $encoding, false);
+    $marker = new HtmlReplaceMarker();
+    $this->insertSnippet($dom);
+    $this->removeWovnIgnore($dom, $marker);
```

Comment:
For Unified Values support, I am not sure we can completely remove `wovn-ignore` content.
@kyuden can you confirm.

Reply:
@kyuden please check

Reply:
In current implementation of swapper, wovn-ignore is needed. original content is key to find the destination.

Reply:
I talk with @mrasu in person. we don't need wovn-ignore content. but wovn-ignore tag is needed.

Reply:
ok done.